esmvaltool/recipes/recipe_flato13ipcc.yml

```
@@ -50,6 +51,35 @@ preprocessors:
       statistics: [mean]
       exclude: [reference_dataset, alternative_dataset]
 
+  zonal_mean:
+    custom_order: True
```

```suggestion
    custom_order: true
```
please use `yamllint` to check your yaml